docs/reference/configuration/Storage.md

```diff
@@ -8,4 +8,91 @@ Name: filesystem.present
     Defined by: library:filesystem
```

Comment:
The new config options make the previous value seem out of place. Is that intended?

Reply:
Does it seem more of a contrast than the [connectivity config page](https://github.com/ARMmbed/Handbook/blob/new_engine/docs/reference/configuration/Connectivity.md), which goes from ppp-cell-iface to lwip? If so, how do you recommend we fix it?

Reply:
That is a different problem probably best solved with a lot of #####################################################################################################################################################################################################################################################################
and some text between the sections that dont correspond. in this case I'm referring to filesystem.present and how that relates to littlefs or fatfs

Reply:
Aha. That makes sense. Thanks for clarifying.

Reply:
I'll fix the connectivity config page in a separate PR. For this one, how do you recommend we show the relationship between filesystem.present and littlefs or fatfs?